README.md

```diff
@@ -29,8 +29,9 @@ The goal of this project is to provide a quickstart for creating nOS dApps with
 
 
 ## Installation
+Before we start, we need to install npm and in order to avoid permission issues on different OS, we suggest using nvm. You can follow the install instructions [here](https://docs.npmjs.com/getting-started/installing-node#using-a-version-manager-to-install-nodejs-and-npm).
```

Comment:
Before we start, we want to make sure npm is installed via a version manager. To avoid permission issues on different OS, we suggest using nvm.

Reply:
Done